tests/fixtures/major-name-modification.json

```diff
@@ -50,10 +52,35 @@
         "type": "Feature",
         "properties": {
           "name": "BANGALURU",
+          "wikidata": "Q40435",
+          "osm:version": 9
+        },
+        "geometry": null
+      },
+      "oldVersion": {
+        "type": "Feature",
+        "properties": {
+          "name": "Bengaluru",
           "wikidata": "Q40435"
         },
         "geometry": null
       },
+      "expectedResult": {
+        "result:levenshtein_distance": 8,
+        "result:major_name_modification": true
+      }
+    },
+    {
+      "description": "Test minor name modification with case",
```

Comment:
There is already a test case with the description, `Test minor name modification with case`, shall we modify this to `Test major name modification with version > 10`